src/stream.rs

```diff
@@ -52,11 +52,19 @@ pub struct StreamMap {
     /// Peer's maximum unidirectional stream count limit.
     peer_max_streams_uni: u64,
 
+    /// The total number of bidirectional streams opened by the peer.
+    peer_opened_streams_bidi: u64,
+
+    /// The total number of unirectional streams opened by the peer.
```

Comment:
```suggestion
    /// The total number of undirectional streams opened by the peer.
```

Reply:
Ugh, I wonder if I'll ever get this right.